packages/image_picker/image_picker/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+
+## 0.8.4+12
+
+* iOS：support images with different formats
```

Comment:
Please specify all the newly added formats here. Also don't forgot the period in the end of the sentence :).